q2_vsearch/_format.py

```diff
@@ -0,0 +1,21 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2017, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import qiime2.plugin.model as model
+
+
+class UchimeStatsFmt(model.TextFileFormat):
+    def sniff(self):
+        with open(str(self)) as fh:
+            header = fh.readline().strip().split('\t')
```

Comment:
Are there header labels we can match? Or is it just values? If just values, is there any potential for needing the escaping rules of a CSV parser?

Reply:
There are no headers, but I'm not sure about the CSV parser. Should I just adapt it to use that?

Reply:
A CSV parser would probably be better